src/createStore.js

```diff
@@ -62,6 +62,7 @@ export default function createStore(reducer, initialState) {
 
     return function unsubscribe() {
       var index = listeners.indexOf(listener);
+      if (index === -1)return;
```

Comment:
I don't mind either way. :stuck_out_tongue: 
I was kind of a fan of `> -1` for some time, but now I think `!== -1` makes the intention clearer.
But whatever, I'm fine with either.